### ISSUE TITLE
Tags don't appear in feed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#557]
```
 :(
```

Original issue reported on code.google.com by `ericbide...@gmail.com` on 1 Jul 2011 at 12:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Aug 2011 at 6:04
- Added labels: **Milestone-7.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 3 Oct 2011 at 2:56
- Added labels: **redesign**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 603f8a634723.
```

Original comment by `ericbide...@html5rocks.com` on 3 Oct 2011 at 7:28
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 603f8a634723.
```

Original comment by `ericbide...@html5rocks.com` on 11 Oct 2011 at 11:17
